spec/models/completed_lesson_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe CompletedLesson, type: :model do
+  context 'associations' do
+    it { is_expected.to belong_to :user }
+    it { is_expected.to belong_to :lesson }
+  end
+
+  describe "validates" do
+    it("should have a default value of completed as true") do
+      # GIVEN 
+      # A new instance of a Completed Lesson without specifying the boolean value
+      user = FactoryBot.create(:user, id:1)
+      lesson = FactoryBot.create(:lesson, id:1)
+      completed_lesson = FactoryBot.create(:completed_lesson, user_id: "1", lesson_id: "1")
+      
+      # WHEN 
+      # Validations are triggered. 
+      completed_lesson.valid?
+
+      # THEN
+      # We expect the boolean value of "completed" to be "true"
+      expect(completed_lesson.completed?).to(eq(true)) 
```

Comment:
The test is passing because you set `completed: true` in the `completed_lesson` factory. To ensure the value is changing to true, the `GIVEN` step should be
```ruby
completed_lesson = FactoryBot.nuild(:completed_lesson, completed: nil)
```

This sets `completed` to `nil`, so now you can safely test that after calling `valid?` completed is set to true.

Also, note that i changed `create` to `build`. `build` does not save the object, and saving the object triggers the validation, which affects your test steps, as you rely on manually triggering `valid?`